tests/test_europy.py

```diff
@@ -1,8 +1,9 @@
 from pandas import DataFrame
 
-from europy.decorators import bias, fairness, unit, minimum_functionality, integration, data_bias, test, \
-    decorator_factory
-from europy.lifecycle.result import TestLabel
+import europy
+from europy.lifecycle.result import TestResult, TestLabel
+from europy.lifecycle.report import Report
+from europy.decorators import *
```

Comment:
hmm, in javaland doing an import * is kind of taboo but I am unsure that there are any consequences in python

Reply:
we are using every decorator in this test file. We can explicit import each, doesn't matter. Nice to use * while developing so you don't have to remember to import. Remember this is just a alpha release.. I'll add a todo here.